src/hrtf/reader.c

```diff
@@ -163,11 +257,11 @@ static struct MYSOFA_HRTF *getHrtf(struct READER *reader, int *err) {
 				*err = getDimension(&hrtf->E, &dir->dataobject);
 				dimensionflags |= 8;
 				break;
-			case 'N':
+			case 'N':			       
 				*err = getDimension(&hrtf->N, &dir->dataobject);
 				dimensionflags |= 0x10;
 				break;
-			case 'M':
+			case 'M':			       
```

Comment:
please run clang-format-8 before commiting

Reply:
Any hint about the specific options ? I ran `clang-format --style="{IndentWidth: 8}" ` (which I assume is equivalent ?) and it reformatted a lot of code. There's no binary called `clang-format-8` on my system.